openquake/commonlib/nrml_examples/source_model/mixed.xml

```diff
@@ -123,6 +123,22 @@ xmlns:gml="http://www.opengis.net/gml"
             <rake>
                 30.0
             </rake>
+            <hypoList>
+                <hypo alongStrike="0.25" downDip="0.25" weight="0.3"/>
+                <hypo alongStrike="0.75" downDip="0.75" weight="0.7"/>
+            </hypoList>
+            <slipList>
+                <slip
+                weight="0.7"
+                >
+                    90.0
+                </slip>
+                <slip
+                weight="0.3"
+                >
+                    135.0
+                </slip>
+            </slipList>
```

Comment:
I find the format adopted for the two list not fully consistent, but I'm sure you did it for specific reasons.